README.rst

```diff
@@ -352,6 +352,13 @@ The list of IPv6 CIDR blocks of the subnet in which the interface resides, e.g.
 ``['2001:db8:abcd:ef00::/64']``. If the subnet does not have any IPv6 CIDR
 blocks or the instance isn't in a VPC, the list will be empty, e.g. ``[]``.
 
+``tags: map``
+~~~~~~~~~~~~~~~~~~~
+
+The map of instance tags when it is enabled, e.g.
+``{'Name': 'my-instance'}``.
+If this is not enabled, it returns an empty map, e.g. ``{}``.
```

Comment:
Returning an empty dict when tags are unavailable seems like a dangerous default. It makes it impossible to tell if the instance has no tags, or if they're just not available.

I considered making `None` the default, but I settled on allowing the attribute to raise an error. This will make it easier to use - no need to check if it's `None` before accessing tag values.

Reply:
Whilst testing I discovered a bug where EC2 doesn't return 404 after you enable then disable the feature - it instead returns an empty response 🤷‍♂️ I've documented this.